packages/ncids-twig/components/nci-guide-card/nci-guide-card-group.twig

```diff
@@ -6,13 +6,13 @@
 				<li class="nci-guide-card desktop:grid-col-4">
 					<div class="nci-guide-card__wrapper">
 						<div class="nci-guide-card__body">
-							<h2 class="nci-guide-card__title">{{ card.title }}</h2>
-							<ul class="nci-card__button-group">
+							<h2 id="{{ card.title|replace({' ': ''}) }}" class="nci-guide-card__title">{{ card.title }}</h2>
```

Comment:
have to worry about more than just spaces, you can use a built in filter instead 

https://twig.symfony.com/doc/3.x/filters/url_encode.html

Reply:
@olitharp-nci this should be addressed now, good shout !